dockerfiles/wiremock/__files/custom_domain_verification/cd_auth0managed.json

```diff
@@ -0,0 +1,19 @@
+{
+  "custom_domain_id": "cd_auth0managed",
+  "domain": "terraform-provider.auth0.com",
+  "primary": true,
+  "status": "{{parameters.status}}",
+  "type": "auth0_managed_certs",
+{{#eq parameters.status "ready"}}
```

Comment:
I enabled response templating to make this work, but I could see it being confusing -- let me know if you want me to back this out and go back to separate files for each status.

Reply:
Yeah, that's what I was worried about too. I'll split it back out into a couple of files.